src/components/home/components/Home.js

```diff
@@ -26,6 +27,7 @@ class Home extends React.Component {
                         </Col>
                     </Row>
                 </div>
+                <CarouselImages />
```

Comment:
remove and add this component to the `productList` component. @joshuaobinna  guide him properly as we want it to be.

Reply:
@MrCEO63 when a user navigates to the maket section,  the carousel should be seen just in between the navbar and the products displayed.  Dm me for more clarifications.